responsive.css

```diff
@@ -0,0 +1,27 @@
+@media screen and (max-width: 680px) {
+  .post__header__logo {
+    height: 40px;
+    margin: 0 20px;
+    border-bottom: none;
+  }
+  .post__header {
+    position: absolute;
+    top: 0;
+    bottom: 60%;
+    margin-bottom: 5px;
+    width: 100%;
+  }
+  .post__content__header {
+    padding: 40px;
+    border-bottom: none;
+  }
+  .post__content {
+    position: absolute;
+    top: 10%;
+    width: 100%;
+    margin-top: 20%;
+    margin-left: 0;
+    margin-right: 0;
```

Comment:
there's a lot of margin/top/padding going on here. I would try to keep this to a minimum. If it's absolutely positioned, try your best to get it right with top/bottom/right/left.

I actually don't think you even need margin-left and margin-right here